packages/p2p/src/components/my-ads/buy-ad-payment-methods-list.jsx

```diff
@@ -0,0 +1,205 @@
+                        <Formik enableReinitialize initialValues={{ payment_method: '' }}>
+                            {({ setFieldValue }) => (
+                                <Field name='payment_method'>
+                                    {({ field }) => (
+                                        <div className='p2p-my-ads--border'>
+                                            <Autocomplete
+                                                {...field}
+                                                autoComplete='off' // prevent chrome autocomplete
+                                                className='quick-add-modal--input'
+                                                data-lpignore='true'
+                                                is_alignment_top
+                                                leading_icon={<Icon icon='IcAddOutline' size={14} />}
+                                                list_items={payment_methods_list}
+                                                list_portal_id='deriv_app'
+                                                onItemSelection={({ value }) => onClickPaymentMethodItem(value)}
+                                                onBlur={e => {
+                                                    e.preventDefault();
+                                                    setFieldValue('payment_method', '');
+                                                }}
+                                                placeholder={localize('Add')}
+                                                required
+                                                trailing_icon={<></>}
+                                                type='text'
+                                            />
+                                        </div>
+                                    )}
+                                </Field>
+                            )}
+                        </Formik>
+                    )}
+            </div>
+        );
+    }
+
+    return (
+        <div className='buy-ad-payment-methods-list__container'>
+            <Formik enableReinitialize initialValues={{ payment_method: '' }}>
+                {({ setFieldValue }) => (
+                    <Field name='payment_method'>
+                        {({ field }) => (
+                            <div className='p2p-my-ads--border'>
+                                <Autocomplete
+                                    {...field}
+                                    autoComplete='off' // prevent chrome autocomplete
+                                    className='quick-add-modal--input'
+                                    data-lpignore='true'
+                                    is_alignment_top
+                                    leading_icon={<Icon icon='IcAddOutline' size={14} />}
+                                    list_items={payment_methods_list}
+                                    list_portal_id='deriv_app'
```

Comment:
Added `list_portal_id` as `deriv_app` as it is required to align the dropdown list to the top:
<img width="474" alt="Screenshot 2022-06-10 at 3 16 14 PM" src="https://user-images.githubusercontent.com/103016120/173020600-acdb9b9f-6e74-4a14-b3e5-22eb1b7cb4e2.png">
<img width="668" alt="Screenshot 2022-06-10 at 3 43 10 PM" src="https://user-images.githubusercontent.com/103016120/173020633-e6dcc573-853a-44e8-a07d-e7e21fa757c8.png">